platform/README.md

```diff
@@ -0,0 +1,73 @@
+Guice Platform Bill of Materials (BOM)
+============================
+This BOM is an extension of of the Guice BOM. It appends all dependencies needed to run Guice and its extensions.
```

Comment:
I don't quite get the difference between the BOM and platform BOM, I think it might confuse people having two BOMs.